wgpu-core/src/device/mod.rs

```diff
@@ -1029,6 +1030,9 @@ impl<A: HalApi> Device<A> {
                         error,
                     })?;
             }
+            if entry.visibility.contains(wgt::ShaderStages::VERTEX) {
+                required_downlevel_flags |= wgt::DownlevelFlags::VERTEX_ACCESSABLE_STORAGE_BUFFERS;
```

Comment:
would it make sense to call it `VERTEX_READABLE_STORAGE_BUFFERS`?

Reply:
Imo no, cause that would imply that they can be something else (i.e. write only)